CHANGELOG.md

```diff
@@ -113,41 +113,41 @@ COMPATIBILITY:
 
 CONTAINER IMAGES:
 
-- Control plane: `ghcr.io/nginxinc/nginx-kubernetes-gateway:0.4.0`
+- Control plane: `ghcr.io/nginxinc/nginx-gateway-fabric:0.4.0`
```

Comment:
Do we want to change the old container image references? Are we planning on retagging and pushing the old images?

Reply:
No we're not, thanks for catching.